src/app/common/helpers.ts

```diff
@@ -1,7 +1,14 @@
 import dayjs from 'dayjs';
+import advancedFormat from 'dayjs-ext/plugin/advancedFormat';
```

Comment:
@gildub sorry I missed this before, you were importing from the wrong package here. I made this change locally and it works.

```suggestion
import advancedFormat from 'dayjs/plugin/advancedFormat';
```

Reply:
Thank you, I've mended it.